Fluid/Filters/StringFilters.cs

```diff
@@ -145,27 +145,34 @@ public static FluidValue Strip(FluidValue input, FilterArguments arguments, Temp
 
         public static FluidValue Truncate(FluidValue input, FilterArguments arguments, TemplateContext context)
         {
-            string text = input.ToStringValue();
-            int length = Convert.ToInt32(arguments.At(0).ToNumberValue());
+            var ellipsis = "...";
```

Comment:
`const` at the class level

Reply:
I already did that at the beginning, but this will change at the future. We can add an ellipses as parameter